js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -91,6 +91,7 @@ export const AssetGraphExplorer = (props: Props) => {
     useAssetGraphData(props.explorerPath.opsQuery, {
       ...props.fetchOptions,
       computeKinds: props.assetFilterState?.filters.computeKindTags,
+      needsFullData: true,
```

Comment:
how big of a lift would it be to use composition to layer in the extra data requirements? the bool toggle seems like it could be a gnarly foot gun

Reply:
I think I can make non foot-gunny with a function overload type signature.

Reply:
+1, if this is really the only callsite where we need `true`, we could move the two useMemo's up to line 97 here I think, or make a separate hook!